packages/react-components/lib/tree-button-group.tsx

```diff
@@ -55,3 +55,5 @@ export const TreeButtonGroup = (props: TreeButtonGroupProps) => {
     </div>
   );
 };
+
+export default TreeButtonGroup;
```

Comment:
The buttons are pretty explicit so I wouldn't call them something generic like `TreeButtonGroup`. Since it deals with managing "snapshots", we could name it something like `SnapshotControlButtonGroup`.

Other things to consider.
* Forwarding refs.
* Extending and forwarding other props to the root div.
* rename `handle*` to `on*`, following conventions.
* add a `save` button and a prop to control which buttons are shown.